public/css/default.css

```diff
@@ -154,9 +154,8 @@ div.testStep input, div.testStep select {
     min-width: 100px;
 }
 
-.chosen-container-multi .chosen-choices li.search-field input[type="text"]
-    {
-    padding: 0px;
+.chosen-container-multi .chosen-choices li.search-field input[type="text"] {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the name of this overqualified element. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=css%3Aoverqualified-elements)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Simplify this over-specified selector. Maximum allowed depth: 3. Actual depth: 4 [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=css%3Aoverspecific-selectors)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Wrap this string with single quotes instead of double quotes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=css%3Asingle-quotes)